code/UI/MainMenu/CustomizeMenu/CustomizeMenu.razor

```diff
@@ -28,11 +31,12 @@
                     var index = i;
                     <div style="width: 50%; flex-direction: column; font-size: 30px; margin-bottom: 5px;">
                         <label>@($"Grub #{i + 1}")</label>
-                        <TextEntry
-                        text=@player.SelectedGrubNames[i]
-                        MaxLength=@(10)
+                        <TextEntry 
+                        text=@player.SelectedGrubNames[i] 
+                        MaxLength=@(10) 
                         CharacterRegex="^[a-zA-Z0-9 ]+$"
-                        OnTextEdited=@((string text) => { player.SelectedGrubNames[index] = text; player.SerializeGrubNames(); }) />
+                        OnTextEdited=@((string text) => { player.SelectedGrubNames[index] = text;
```

Comment:
Can you disable the vscode formatter, it is throwing everything onto a second line when everything gets too long.